### PR TITLE
chore: enforce files naming to use .styles.ts [cxe-coastal]

### PR DESCRIPTION
**Pull request by layershifter**
## New Behavior

This PR removes `useCOMPONENTStyles.ts` files to `useCOMPONENTStyles.styles.ts` files to follow [Griffel's rule for naming files](https://github.com/microsoft/griffel/blob/main/packages/eslint-plugin/src/rules/styles-file.md).

The ESLint rule have not been enabled yet, this PR is a preparation to do that. Affects files owned by @microsoft/cxe-coastal 

Change files are set to patch/prerelease as artifacts should be updated.

## Related Issue(s)

- Related to #27672

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Avatar-mount_base.html">724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Avatar-mount.html">701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Button-mount_base.html">373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Button-mount.html">374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Field-mount_base.html">1284</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Field-mount.html">1284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProvider-mount_base.html">894</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProvider-mount.html">922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-mount_base.html">120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-mount.html">121</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">93</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">95</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">99</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">100</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/InfoButton-mount_base.html">20</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/InfoButton-mount.html">22</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/MakeStyles-mount_base.html">1125</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/MakeStyles-mount.html">1135</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Persona-mount_base.html">1998</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/Persona-mount.html">1953</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/SpinButton-mount_base.html">1568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27708/perf-test-react-components/SpinButton-mount.html">1605</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           | `86.012 kB`<br />`26.086 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 | `87.199 kB`<br />`28.095 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 | `85.583 kB`<br />`27.692 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `65.31 kB`<br />`17.931 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `204.34 kB`<br />`57.141 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `36.086 kB`<br />`11.9 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |  `17.399 kB`<br />`6.298 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `6.446 kB`<br />`2.186 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |  `25.357 kB`<br />`8.798 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    | `34.112 kB`<br />`11.018 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bad96553b2d34ffb3d45c32a44d4883046a15945'>bad96553b2d34ffb3d45c32a44d4883046a15945</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-select/src/components/Select/useSelectStyles.styles.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ const useRootStyles = makeStyles({
   },
 });
 
-const useSelectStyles = makeStyles({
+const useSelectStylesStyles = makeStyles({
```
Comment (khmakoto) [inline packages/react-components/react-select/src/components/Select/useSelectStyles.styles.ts:107]: Why are we making this change?

Comment (layershifter) [reply packages/react-components/react-select/src/components/Select/useSelectStyles.styles.ts:107]: Oops, not intentional, will revert

**Review by khmakoto — APPROVED** [review]

**Pull request merged**